rust/kernel/prelude.rs

```diff
@@ -11,16 +11,22 @@
 //! use kernel::prelude::*;
```

Comment:
Typo in the commit message: ofç

Reply:
Thanks for actually reading the commit messages even for a cleanup like this. Very much appreciated.